api/src/routes/productRouter.js

```diff
@@ -0,0 +1,14 @@
+const { Router } = require('express');
+const router = Router();
+
+router.use('/getAll',(req,res)=>{
```

Comment:
Change this line to check if the query exists or not, then apply logic based on that